src/js/components/charts/Chart.js

```diff
@@ -24,6 +25,12 @@ var Chart = React.createClass({
 
   componentWillMount: function () {
     this.internalStorage_set({width: null});
+    this.store_listeners = [
```

Comment:
actually since this is the old type of component, you can add it on directly to the object (similar to how you do `mixins: [etc, etc]`